data/lua/wml/objectives.lua

```diff
@@ -95,7 +97,9 @@ local function generate_objectives(cfg)
 					lose_objectives = lose_objectives .. caption .. "\n"
 				end
 
-				lose_objectives = lose_objectives .. color_prefix(r, g, b) .. objective_bullet .. description .. turn_counter .. "</span>" .. "\n"
+				lose_objectives = lose_objectives
+					.. color_prefix(r, g, b) .. objective_bullet.. description
+					.. turn_counter .. "</span>" .. "\n"
```

Comment:
Is it okay to do these with the operator on the next line? It doesn't work in the REPL, though that doesn't mean it won't work in a file...

(To be clear, I'm referring to this:
```
lose_objectives = lose_objectives
    .. stuff
```
compared to this:
```
lose_objectives = lose_objectives ..
    stuff
```
)

Side note, one of your `..` here is missing a preceding space.

Reply:
It's easier to see the operation if it is given prime placement, so I like it on a new line.

But, please, be consistent!

If you're going to break it up break it all up.


```
    lose_objectives = lose_objectives
                    .. color_prefix(r, g, b)
                    .. objective_bullet
                    .. description
                    .. turn_counter
                    .. "</span>"
                    .. "\n"
```

Reply:
I don't like aligning to match, though...

Reply:
@CelticMinstrel I personally prefer to keep concatenation (or other) operators on new lines so the text would feel more natural if you read left-to-right.

Like, this is how my eyes will read text at first:

```lua
val some_string = "this is a very long str............too..long...don't...read...here "
    .. "and also has a .........again...too.....far....away"
```

So I see the most important operator first (`..`), not on column 98.
Also, if you forget `..`, your Lua code will compile (ouch!). So some visibility for `..` is almost required.

Reply:
Plus, it's more maintainable. You can add, remove, and re-arrange a lot more easily.

Reply:
I'm not sure about splitting each string concatenation on a new line. @CelticMinstrel, what do you think? For example, in this code:

```lua
    lose_objectives = lose_objectives
                    .. color_prefix(r, g, b)
                    .. objective_bullet
                    .. description
                    .. turn_counter
                    .. "</span>"
                    .. "\n"
```

The _meaning_ of the text is show `lose_objectives` first, then show rest of the info inside `<span></span>`  (the opening `<span>` is encoded in `color_prefix(r,g,b)`).

Reply:
Then again, I have nothing really serious against splitting totally, just a matter of preference in this case. Will do once any new vote on this appears and forget about it.

Reply:
I'm not sure... I guess it's not terrible for every element to be on a new line, but when each element is quite short, it seems a little silly.

Also, I don't like indenting to align with the `=`. Just indent by a tab or four spaces (I forget which was the standard for Lua files in Wesnoth).

Reply:
I prefer a tab, too. OK, will leave these two lines as is for now then.

Reply:
EDIT: ^